lib/remoteproc/remoteproc.c

```diff
@@ -900,6 +900,16 @@ static int remoteproc_virtio_notify(void *priv, uint32_t id)
 	return 0;
 }
 
+static int remoteproc_virtio_wait_notified(void *priv, uint32_t id)
```

Comment:
To use `remoteproc` prefix instead of `remoteproc_vitio`
-> `remoteproc_wait_notified()`

Reply:
Done, and should I change the `remoteproc_virtio_notify` to `remoteproc_notify` too?

Reply:
Yes if you want, but in a new PR, please